js/router.js

```diff
@@ -121,17 +85,19 @@ export default class ObRouter extends Router {
       pageOpts.layer = args[1];
     }
 
-    this.getUser(guid).done((user) => {
+    const profile = new Profile({ id: guid });
+
+    profile.fetch().done(() => {
```

Comment:
I also eliminated the dummy user data collection, so the user page route is actually fetching from the server.